src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -605,6 +553,21 @@ Metric retention periods are not editable.
     </table>
   </Collapser>
 
+  <Collapser
+    id="browser-namespaces"
+    title="Browser namespaces"
+  >
+
+All browser namespaces have the same default retention period. Details on the browser namespaces and the events they report: 
```

Comment:
Can we add something small to the second part, like "Here are some details on the browser namespaces..." or something along those lines? What's here is fine it just feels a little robotic when I read it. Just a small thing.